themes/vocabulary_theme/templates/layout.html

```diff
@@ -41,7 +41,7 @@
 </script>
 <title>{% block title %}Welcome{% endblock %} — Creative Commons Open Source</title>
 
-<body>
+<body style="overflow-x: hidden;">
```

Comment:
@TimidRobot @Cronus1007 should I work towards the issue further now?

Reply:
@TimidRobot I found out why there is an overflow, so it's because there is an extra margin right in div.columns under the hero section. Removing the margin-right fixes it. 

Should I do the change inline in the hero.html file or in style.css inside the assets folder?
In style.css, adding the following fixes the problem without affecting anything else:

.hero .container .columns{
  margin-right: 0;
}

Reply:
@TimidRobot

Hi, sorry for the delayed response.

So, adding margin-right-0 class to div. columns fix the issue.

Should I make a pull request?

Reply:
I am unable to figure out how to do that. Can I fork the main repository again, and make another PR?